docs/aced-RBAC.md

```diff
@@ -0,0 +1,379 @@
+            - manchester_requestor_updater_role
+          resource_paths:
+            - /programs/manchester
+
+        - id: stanford_requestor_updater
+          description: Allows approving access to any resource under "/programs/stanford"
+          role_ids:
+            - stanford_requestor_updater_role
+          resource_paths:
+            - /programs/stanford
+
+
+        - id: requestor_reader
+          role_ids:
+            - requestor_reader_role
+          resource_paths:
+            - /programs
+            - /sower      
+
+        # jobs (sower)
+        - description: be able to use sower job
+          id: sower
+          resource_paths: [/sower]
+          role_ids: [sower_user]
+
+
+        roles:
+
+        - id: writer
+          description: Allows all write actions in a project
+
+          permissions:
+
+          - id: file_upload
+            description: upload files to a project
+            action:
+              service: fence
+              method: file_upload
+
+          - id: storage_writer
+            action:
+              service: '*'
+              method: write-storage
+
+          - id: creator
+            action:
+              service: '*'
+              method: create
+
+          - id: updater
+            action:
+              service: '*'
+              method: update
+
+          - id: deleter
+            action:
+              service: '*'
+              method: delete
+
+
+
+        - id: reader
+          description: Allows all read actions in a project
+
+          permissions:
+
+          - id: storage_reader
+            action:
+              service: '*'
+              method: read-storage
+
+          - id: reader
+            action:
+              service: '*'
+              method: read
+
+        - id: workspace_user
+          permissions:
+          - id: workspace_access
+            action:
+              service: jupyterhub
+              method: access
+
+        - id: administrator
+          description: full access to all services
+          permissions:
+
+          - id: all
+            action:
+              service: '*'
+              method: '*'
+
+          - id: sheepdog_admin_action
+            description: CRUD access to programs and projects
+            action:
+              service: sheepdog
+              method: '*'
+
+          - id: indexd_admin
+            description: full access to indexd API
+            action:
+              service: indexd
+              method: '*'
+
+
+        # REQUESTOR ROLES
+        - id: requestor_creator_role
+          permissions:
+            - id: requestor_creator_action
+              action:
+                service: requestor
+                method: create
+
+        - id: requestor_updater_role
+          permissions:
+            - id: requestor_updater_action
+              action:
+                service: requestor
+                method: update
+
+
+        - id: ohsu_requestor_updater_role
```

Comment:
you could use `requestor_updater_role` everywhere, it's identical to `ohsu_requestor_updater_role`, `ucl_requestor_updater_role`, etc

Reply:
ah.  nice catch.  thank you.